cli/cmd/update_project.go

```diff
@@ -118,29 +114,48 @@ keptn update project PROJECTNAME --git-user=GIT_USER --git-remote-url=GIT_REMOTE
 				return errors.New("Proxy cannot be set without scheme")
 			}
 
-			project.GitProxyURL = *updateProjectParams.GitProxyURL
-			project.GitProxyScheme = *updateProjectParams.GitProxyScheme
-			project.GitProxyUser = *updateProjectParams.GitProxyUser
-			project.GitProxyPassword = *updateProjectParams.GitProxyPassword
-			project.InsecureSkipTLS = *updateProjectParams.InsecureSkipTLS
+			project.GitCredentials = &apimodels.GitAuthCredentials{
```

Comment:
there seems to be quite some code duplication when it comes to setting the git credentials config if we look ad `update_project` and `create_project` - can we extract this part into a separate function and use it in both commands?